src/cli/commands/upgrade-interactive.js

```diff
@@ -40,21 +40,19 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
   const maxLengthArr = {
     name: 'name'.length,
     current: 'from'.length,
-    range: 'range'.length,
+    range: 'latest'.length,
     [outdatedFieldName]: 'to'.length,
   };
 
-  if (deps.length === 0) {
-    reporter.success(reporter.lang('allDependenciesUpToDate'));
-    return;
-  }
```

Comment:
I don't think you meant to delete this :)

Reply:
Oh. Sorry for that, will fix in next commit.

Reply:
Also moved the check before the creation of `maxLengthArr` to return as quickly as possible